js/packages/web/src/views/auction/index.tsx

```diff
@@ -202,6 +205,28 @@ const BidLine = (props: { bid: any; index: number; mint?: MintInfo, isCancelled?
   const bidder = bid.info.bidderPubkey.toBase58();
   const isme = wallet?.publicKey?.toBase58() === bidder;
 
+  // Get Twitter Handle from address
+  const connection = useConnection();
+  const [bidderTwitterHandle, setBidderTwitterHandle] = useState('');
+  useEffect(() => {
+    const getTwitterHandle = async (
+      connection: Connection,
+      bidder: PublicKey,
+    ): Promise<string | undefined> => {
+      try {
+        const [twitterHandle] = await getHandleAndRegistryKey(
```

Comment:
does this one use getMultipleAccounts? I want to make sure we are not blowing up rate limits?

Reply:
Sorry I just read this, will check on that.

Reply:
FYI @bartosz-lipinski 
https://github.com/solana-labs/solana-program-library/blob/master/name-service/js/src/twitter.ts#L245

Reply:
Yeah that one is using individual getAccountInfo and doesn’t cache. We should query for all visible bids with single call and cache